net/netifyd/Makefile

```diff
@@ -0,0 +1,83 @@
+# Netify Agent
+# Copyright (C) 2016-2018 eGloo, Incorporated
+#
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=netifyd
+PKG_VERSION:=2.67
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Darryl Sokoloski <darryl@egloo.ca>
+PKG_LICENSE:=GPL-3.0+
+
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+PKG_CONFIG_DEPENDS:=CONFIG_LIBCURL_ZLIB
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.com/netify.ai/public/netify-agent.git
+PKG_SOURCE_DATE:=2018-07-04
+PKG_SOURCE_VERSION:=d05494e074495909837186bd7fad11ba89022d92
+PKG_MIRROR_HASH:=9ed3232739d162985a4f6f6e156792a0bed572f1a52a4618cb96604c0b0c21bf
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/netifyd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Netify Agent
+  URL:=http://www.netify.ai/
+  DEPENDS:=+libcurl +libmnl +libnetfilter-conntrack +libjson-c +libpcap +zlib +libpthread +libstdcpp
```

Comment:
Can we replace libstdcpp with `$(CXX_DEPENDS)` from `$(INCLUDE_DIR)/uclibc++.mk`?

Also, why there is no ndpi listed there?  Are you doing static linking?

Reply:
@yousong I answered this question previously (@diizzyy make the same observation), and after testing, discovered that `<unordered_map>` is not available for uclibcxx:
> Unfortunately by doing this, I loose support for <unordered_map> which is essential for performance reasons. I require libstdcpp for this.

> Also, why is there no ndpi listed here? Are you doing static linking?

Yes, we include a heavily modified nDPI submodule that we forked and maintain here:
https://gitlab.com/netify.ai/public/netify-agent-libs/ndpi

We contribute fixes and useful changes back to nDPI but we make so many other (drastic) changes to the library that there is no sensible way to merge (they wouldn't want most of our changes).

Reply:
@dsokoloski , please consider adding some comment lines for possible future readers of the code, especially these corner cases that are this package specific.